src/main/java/uk/gov/hmcts/cmc/claimstore/services/legaladvisor/DocAssemblyClient.java

```diff
@@ -0,0 +1,137 @@
+package uk.gov.hmcts.cmc.claimstore.services.legaladvisor;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDHearingCourtType;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDOrderDirection;
+import uk.gov.hmcts.cmc.ccd.domain.legaladvisor.CCDOrderDirectionType;
+import uk.gov.hmcts.cmc.claimstore.idam.models.UserDetails;
+import uk.gov.hmcts.cmc.claimstore.processors.JsonMapper;
+import uk.gov.hmcts.cmc.claimstore.services.UserService;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+
+import java.time.LocalDate;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Component
+public class DocAssemblyClient {
+    private static final String TEMPLATE_NAME = "CV-CMC-GOR-ENG-0004.docx";
```

Comment:
Can we please not hard code template name here? please out in properties instead.